Library/Homebrew/livecheck/livecheck.rb

```diff
@@ -268,7 +269,6 @@ def run_checks(
         end
 
         current_str = current.to_s
-        current = LivecheckVersion.create(formula_or_cask, current)
```

Comment:
I don't think this should be removed – is there any particular reason to do so?

Reply:
Oops! my bad. I removed it because `brew typecheck` was giving me a warning here, but I guess it was before all the previous changes that I have done. Now, there's no issue with it.